src/pcre2test.c

```diff
@@ -9109,9 +9122,6 @@ for (gmatched = 0;; gmatched++)
 
   if ((dat_datctl.control & (CTL_FINDLIMITS|CTL_FINDLIMITS_NOHEAP)) != 0)
     {
-    capcount = 0;  /* This stops compiler warnings */
```

Comment:
I've looked back to the code in 2017 when Philip added this.

In that version, `capcount` was assigned conditionally, so it would indeed have generated a warning. The suppression makes no sense anymore though, because we assign to `capcount` unconditionally just a line or two below. This is a "dead store", a value we put in the variable and never read back.